handler/src/test/java/io/netty/handler/ssl/JdkSslEngineTest.java

```diff
@@ -271,6 +273,22 @@ public void testEnablingAnAlreadyDisabledSslProtocol() throws Exception {
         testEnablingAnAlreadyDisabledSslProtocol(new String[]{}, new String[]{PROTOCOL_TLS_V1_2});
     }
 
+    @Ignore /* Does the JDK support a "max certificate chain length"? */
+    @Override
+    public void testMutualAuthValidClientCertChainTooLongFailOptionalClientAuth() throws Exception {
+    }
+
+    @Ignore /* Does the JDK support a "max certificate chain length"? */
+    @Override
+    public void testMutualAuthValidClientCertChainTooLongFailRequireClientAuth() throws Exception {
+    }
+
+    @Override
+    protected boolean mySetupMutualAuthServerIsValidException(Throwable cause) {
+        // TODO(scott): work around for a JDK issue. The exception should be SSLHandshakeException.
```

Comment:
@normanmaurer - IIRC you opened an issue against the JDK? Can we create a Netty issue with the JDK issue and a link to this PR to cleanup these TODOs